modules/ROOT/pages/on-error-scope-concept.adoc

```diff
@@ -60,18 +60,61 @@ to fatal errors, which have error messages that contain the word "fatal":
 ----
 <on-error-continue
   enableNotifications="true" logException="true"
-  doc:name="On Error Continue" type="ANY"
+  type="ANY"
   when='error.cause.message contains "fatal"'/>
 ----
 
 Note that matching conditions are evaluated sequentially, in the order in
-which the On-Error components reside in the Error Handler. So, for example, if
-you want to handle specific HTTP errors in one way but handle all other errors
-a different way, you need to provide an On-Error component configuration for
-the HTTP errors before a component that captures the remaining errors with
-ANY. Keep in mind that default error handling takes place for errors you
-that you do not explicitly match or use ANY to capture. The default error
-handler logs and propagates such unhandled errors.
+which the On-Error components reside in the Error Handler. For example, if
+you want to handle `VALIDATION:NOT_NULL` in one way but handle all other
+errors a different way, you need to provide an On-Error component configuration
```

Comment:
Most instances of "you need to" are not necessary, although they might sound better to our ears. It is more active and forceful to omit it when possible:
```suggestion
errors a different way, provide an On-Error component configuration
```